corehq/apps/userreports/models.py

```diff
@@ -778,7 +785,53 @@ def ui_filters(self):
     @property
     @memoized
     def charts(self):
```

Comment:
It is used in data preview and while viewing report as well. It's also used for mobile UCRs

Reply:
The description says this will not work on mobile, does this mean we will be sending down the new info but the phone just still won't be able to render that chart? Does anything change on the phone based on this work, or it is broken now and will stay broken after?

Reply:
Before this change does it break as well? I think I am trying to figure out if this change can make any mobile reports worse. If not, definitely fine, if so, we may want to make sure this is at least neutral on mobile before merging

Reply:
https://github.com/dimagi/commcare-hq/pull/30873#issuecomment-1054490315